### ISSUE TITLE
Problem parsing non numeric characters in pages

### ISSUE THREAD

**Issue by guillermomarco** [brechtm/citeproc-py#69]
I'm facing a problem with references containing non numeric characters in `pages` field.

```
  File "/usr/local/lib/python2.7/dist-packages/citeproc/source/bibtex/bibtex.py", line 77, in __init__
    self.add(self.create_reference(key, entry))
  File "/usr/local/lib/python2.7/dist-packages/citeproc/source/bibtex/bibtex.py", line 234, in create_reference
    csl_fields = self._bibtex_to_csl(bibtex_entry)
  File "/usr/local/lib/python2.7/dist-packages/citeproc/source/bibtex/bibtex.py", line 98, in _bibtex_to_csl
    value = self._bibtex_to_csl_pages(value)
  File "/usr/local/lib/python2.7/dist-packages/citeproc/source/bibtex/bibtex.py", line 120, in _bibtex_to_csl_pages
    pages = Pages(first=int(decimal))
ValueError: invalid literal for int() with base 10: 'R106'
```

The article in bib that is causing the problem:

> @article{Anders2010,
> abstract = {High-throughput sequencing assays such as RNA-Seq, ChIP-Seq or barcode counting provide quantitative readouts in the form of count data. To infer differential signal in such data correctly and with good statistical power, estimation of data variability throughout the dynamic range and a suitable error model are required. We propose a method based on the negative binomial distribution, with variance and mean linked by local regression and present an implementation, DESeq, as an R/Bioconductor package.},
> author = {Anders, S and Huber, W},
> doi = {10.1186/gb-2010-11-10-r106},
> isbn = {1474-760X (Electronic)$\backslash$r1474-7596 (Linking)},
> issn = {1465-6914},
> journal = {Genome Biol},
> keywords = {Animals,Binomial Distribution,Chromatin Immunoprecipitation/methods,Computational Biology/_methods,Drosophila/genetics,Gene Expression Profiling/_methods,High-Throughput Nucleotide Sequencing/methods,Linear Models,Models, Genetic,Saccharomyces cerevisiae/genetics,Sequence Analysis, RNA/*methods,Stem Cells,Tissue Culture Techniques},
> number = {10},
> pages = {R106},
> pmid = {20979621},
> title = {{Differential expression analysis for sequence count data}},
> url = {http://www.ncbi.nlm.nih.gov/pubmed/20979621},
> volume = {11},
> year = {2010}
> }

**Comment by guillermomarco** [comment]
#70

**Comment by brechtm** [comment]
Fixed in 87bbe2230aa1d393b71f7d5624031c121c746.
